src/views/Home.vue

```diff
@@ -84,6 +91,67 @@ export default {
 		}
 	},
 	methods: {
+		moveMessage(e) {
+
+			// We don't want to use this.activeAccount here because we want it to work in the unified inbox
+			var startAccount = e.items[0].attributes.href.baseURI.match(/message\/(\d)/)
+			var targetAccount = e.droptarget.href.match(/accounts\/(\d)/)
+
+			if (targetAccount[1] == startAccount[1]) {
+				var accountId = targetAccount[1]
+
+				// We don't want to use this.activeFolder here because we want it to work in the unified inbox
+				var startFolderId = e.items[0].attributes.href.baseURI.match(/.+-([^-]+)-\d+$/)
```

Comment:
why do we actually have to do regex matches here? Isn't there a way to transport the information in JSON and just load the data from the Vuex store?